delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsIndexingRoutes.scala

```diff
@@ -98,7 +98,7 @@ class BlazegraphViewsIndexingRoutes(
                         emit(
                           fetch(id, ref)
                             .flatMap { view =>
-                              projectionErrors.search(view.ref, pagination, timeRange)
+                              projectionErrors.search(view.ref, pagination, timeRange).toUIO
```

Comment:
Could it be `toTask`?

Reply:
Should just be removed actually :) it's hidden by the implicit conversions that basically turn it back